actors/miner/src/expiration_queue.rs

```diff
@@ -947,3 +947,76 @@ fn check_no_early_sectors(set: &BTreeSet<u64>, es: &ExpirationSet) -> anyhow::Re
     }
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use fil_actors_runtime::DealWeight;
+    use fvm_shared::clock::NO_QUANTIZATION;
+    use fvm_shared::sector::StoragePower;
+
+    #[test]
+    fn test_expirations() {
+        let quant = QuantSpec { unit: 10, offset: 3 };
+        let sectors = [
+            test_sector(7, 1, 0, 0, 0),
+            test_sector(8, 2, 0, 0, 0),
+            test_sector(14, 3, 0, 0, 0),
+            test_sector(13, 4, 0, 0, 0),
+        ];
+        let result = group_new_sectors_by_declared_expiration(SectorSize::_2KiB, &sectors, quant);
+        let expected = [
+            SectorEpochSet {
+                epoch: 13,
+                sectors: vec![1, 2, 4],
+                power: PowerPair {
+                    raw: StoragePower::from(2048 * 3),
```

Comment:
nit: if you use `_2KiB` for sector size in the call to `group_by...` then also use it in the storage power calculation here and elsewhere `_2KiB *3`

Reply:
I don't feel that `SectorSize::_2KiB as u64` is better than `2048`, though. Feel free to change it in another PR.

Reply:
oh ok would you mind passing 2048 as the param into `group_new_sectors` too as in the go test then for consistency?  Or do you prefer it with the two different names for some reason?

Reply:
@ZenGround0 `SectorSize` is an enum. It's not interchangeable with a number.

Reply:
ah thx that's where I'm going wrong.  golang brain is hard to shake